### PR TITLE
feat(set-env-vars): add subject, test and solution for the exercise

### PR DESCRIPTION
**Pull request by MSilva95**

### CODE REVIEW

File: sh/tests/solutions/set-env-vars.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+export MY_MESSAGE="Hello World"
+export MY_NUM=100
+export MY_PI=3.142
+export MY_ARR=(one, two, three, four, five)
```
Comment (zamazzal) [inline sh/tests/solutions/set-env-vars.sh:6]: you didn't ask the student to set this env var in the readme file!

File: sh/tests/set-env-vars_test.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# Unofficial Bash Strict Mode
+set -euo pipefail
+IFS='
+'
+
+submitted=$(bash student/set-env-vars.sh)
```
Comment (zamazzal) [inline sh/tests/set-env-vars_test.sh:8]: it's bad idea to compare the all env vars, maybe cause you can't control the output of all of them
i propose to grep by MY_ when you print and ask the student to do that in his script

**Review by zamazzal — CHANGES_REQUESTED** [review]

File: sh/tests/solutions/set-env-vars.sh (hunk 1 above)
Comment (MSilva95) [reply sh/tests/solutions/set-env-vars.sh]: fixed

File: sh/tests/set-env-vars_test.sh (hunk 2 above)
Comment (MSilva95) [reply sh/tests/set-env-vars_test.sh]: Done

**Comment by davhojt** [comment]
Change the title of the PR to match the convention seen here: https://github.com/01-edu/public/pull/1676

**Review by zamazzal — APPROVED** [review]

**Pull request merged**